client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/TruthnessHelper.java

```diff
@@ -0,0 +1,21 @@
+package org.evomaster.client.java.instrumentation.coverage.methodreplacement;
+
+import org.evomaster.client.java.instrumentation.heuristic.Truthness;
+
+public class TruthnessHelper {
+
+    /**
+     * @param len a positive value for a length
+     * @return
+     */
+    public static Truthness getTruthnessToEmpty(int len) {
+        Truthness t;
+        assert (len >= 0);
```

Comment:
in  a `public` method, input validation should rather throw an `IllegalArgumentException`. see discussion on `for_developers.md`

Reply:
sorry, my bad.